packages/babel-cli/src/babel/file.ts

```diff
@@ -40,7 +41,7 @@ export default async function ({
 
       mapSections.push({
         offset: { line: offset, column: 0 },
-        map: result.map || {
+        map: (result.map as EncodedSourceMap) || {
```

Comment:
This cast is unfortunate because the `EncodedSourceMap` in `trace-mapping`
```ts
export interface SourceMapV3 {
    file?: string | null;
    names: string[];
    sourceRoot?: string;
    sources: (string | null)[];
    sourcesContent?: (string | null)[];
    version: 3;
}
export interface EncodedSourceMap extends SourceMapV3 {
    mappings: string;
}
```
is slightly different to the one in `gen-mapping`.

```ts
export interface SourceMapV3 {
    file?: string | null;
    names: readonly string[];
    sourceRoot?: string;
    sources: readonly (string | null)[];
    sourcesContent?: readonly (string | null)[];
    version: 3;
}
export interface EncodedSourceMap extends SourceMapV3 {
    mappings: string;
}
```

Reply:
@jridgewell Maybe you want to align them